platform/source/mbed_retarget.cpp

```diff
@@ -825,7 +825,7 @@ int _lseek(FILEHANDLE fh, int offset, int whence)
 
     off_t off = lseek(fh, offset, whence);
     // Assuming INT_MAX = LONG_MAX, so we don't care about prototype difference
-    if (off > INT_MAX) {
+    if (off > sizeof(int)) {
```

Comment:
Oops - this is now checking if it's bigger than 4. Original code was correct.

Code is trying to be cautious in case off_t is 64-bit, which would allow it to overflow. I don't think there's an easy way to figure out whether off_t is 64-bit or not, and whether the test is necessary. (You could do `#if OFF_T_MAX > INT_MAX` if `OFF_T_MAX` existed, but it doesn't).

I think I'd just suppress the Coverity warning here.

Reply:
Silly mistake.